### PR TITLE
Upgrade HtmlUnit from 2.x to 3.x

### PR DESCRIPTION
**Pull request by timja**
### Desired reviewers

@jenkinsci/chinese-localization-sig

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -184,12 +183,11 @@
     <dependency>
       <groupId>org.jenkins-ci</groupId>
       <artifactId>symbol-annotation</artifactId>
-      <version>1.14</version>
     </dependency>
     <dependency>
       <groupId>io.jenkins.plugins</groupId>
       <artifactId>localization-support</artifactId>
-      <version>1.1</version>
+      <version> 1.1</version>
```
Comment (LinuxSuRen) [inline pom.xml:190]: Extra whitespace?

File: src/main/java/io/jenkins/plugins/localization_zh_cn/CommunityDecorator.java [hunk 2]
```diff
@@ -37,14 +37,8 @@ public boolean showCommunityLink() {
...
-        UserCommunityProperty.ShowConditions condition = UserCommunityProperty.ShowConditions.Chinese;
-        try {
-            String conditionName = communityProperty.getShowCondition();
-            condition = UserCommunityProperty.ShowConditions.valueOf(conditionName);
-        } catch (IllegalArgumentException | NullPointerException e) {
-            // ignore the invalid conditions
-            // when this plugin just be installed without restart, conditionName could be null
-        }
+        String conditionName = communityProperty.getShowCondition();
+        UserCommunityProperty.ShowConditions condition = UserCommunityProperty.ShowConditions.valueOf(conditionName);
```
Comment (LinuxSuRen) [inline src/main/java/io/jenkins/plugins/localization_zh_cn/CommunityDecorator.java:41]: Could you provide the context for these changes? I cannot figure it out by reading the code.

**Review by LinuxSuRen — COMMENTED** [review]
Thanks for the update. A few comments left.

File: src/main/java/io/jenkins/plugins/localization_zh_cn/CommunityDecorator.java (hunk 2 above)
Comment (timja) [reply src/main/java/io/jenkins/plugins/localization_zh_cn/CommunityDecorator.java:41]: It was spotbugs

File: pom.xml (hunk 1 above)
Comment (timja) [reply pom.xml:190]:
```suggestion
      <version>1.1</version>
```

**Review by LinuxSuRen — APPROVED** [review]

**Pull request merged**